pages/getting-started/registration-and-setup.mdx

```diff
@@ -86,25 +107,30 @@ When you’re done filling out the details, your “General” settings should l
   controls
 ></video>
 
-### Build out your newsletter's "Scaffolding"
+This video shows a newsletter's "General" section with information in all the fields.
+
+### Customize your newsletter's design
 
-Your “Header” and “Footer” will appear at the top and bottom (respectively) of every newsletter that you send. These fields can be written out in Markdown, HTML, or plaintext.
+The "Design" section lets you customize the visual elements that appear consistently in every newsletter you send: the header and footer, custom colors or other elements with CSS, and which email template you want to use.
 
-In this section, you also have the option to customize each newsletter’s CSS, or style, with HTML. You can learn more about building out your newsletter’s CSS [here](https://docs.buttondown.email/advanced-features/css).
+The content you put in the “Header” and “Footer” will appear at the top and bottom (respectively) of every newsletter that you send. For example, you might want to put a link to your website in the footer. You can write in Markdown, HTML, or plaintext.
+
+The "CSS" section lets you customize your newsletter’s appearance. To learn more, read [CSS](/advanced-features/css).
 
 <video
   src="/images/settings/registration_scaffolding-settings-section_AdobeExpress.mp4"
   controls
 ></video>
 
+This video shows a newsletter's "Design" section with information in the "Header" and "Footer" sections.
+
 ### Establish your newsletter's "Branding"
 
-Your newsletter’s branding includes its color palette, or “Tint Color,” as well as its logo, or “Icon.” Your “Tint Color” will pop up as an accent in places like hyperlinks or your “Subscribe” button. Your “Icon” will appear on top of your newsletters, in your archives, and as part of your social “Share” button. Upload a square image (ideally a PNG) with a 300 x 300 resolution for the best results.
+Your newsletter’s branding includes its color palette, or “Tint Color,” as well as its logo, or “Icon.” Your “Tint color” appears an accent in places like hyperlinks or the “Subscribe” button. The “Icon” appears on top of your newsletters, in your archives, and as part of your social “Share” button. The "Share image" appears when you or your subscribers share your newsletter on social media.
 
-<video
-  src="/images/settings/registration_branding-settings-section_AdobeExpress.mp4"
-  controls
-></video>
```

Comment:
Removed this video because it shows an outdated UI.

Reply:
Mind deleting the backing video from the repo as well, then?